src/components/sbb-icon/sbb-icon.tsx

```diff
@@ -107,7 +107,7 @@ export class SbbIcon extends LitElement {
   }
 
   protected override render(): TemplateResult {
-    setAttribute(this, 'role', 'img');
+    setAttribute(this, 'role', this.getAttribute('role') ?? 'img');
```

Comment:
Try this, should work the same
```suggestion
    setAttribute(this, 'role', this.role ?? 'img');
```

Reply:
If I change this, the tests on CI will fail 🤷‍♂️

Reply:
mmm maybe I'm missing something. Feel free to choose whether to debug it or leave it as it is